apis/core.oam.dev/common/types.go

```diff
@@ -349,6 +349,8 @@ type WorkflowStep struct {
 
 	If string `json:"if,omitempty"`
 
+	Timeout string `json:"timeout,omitempty"`
```

Comment:
As I commented below, it'll be better if it's a int here.

Reply:
If the value is seconds, it is best to specify this in the key, such as `timeoutSeconds`.

Some references:
[TimeoutSeconds in Kubernetes probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes)
[Argo Timeout](https://argoproj.github.io/argo-workflows/template-defaults/#configuring-templatedefaults-in-workflowspec)
[Argo activeDeadlineSeconds](https://argoproj.github.io/argo-workflows/walk-through/timeouts/)
[Tekton Timeout](https://tekton.dev/vault/pipelines-v0.18.1/tasks/#specifying-a-timeout)
[timeout-minutes in GitHub Action](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes)